### PR TITLE
responsive for services-text-menu

### PR DESCRIPTION
**Pull request by v-nakon**
why responsive don't apply for all text in page?
example for `<div class="slider-title">Work around you and your team</div>`

### CODE REVIEW

**Comment by byzyk** [comment]
Because `slider-title` has fixed width in your case.

**Comment by v-nakon** [comment]
I did the right thing for `services-text-menu`?)

**Comment by v-nakon** [comment]
look, it's right?

**Comment by byzyk** [comment]
image

Add more space on the sides

**Comment by byzyk** [comment]
image

The video is not centrally aligned. I think it's better to remove `absolute` position for mobile devices at all since it's not need there. What do you think?

Also, when resizing the window there is an animation of video moving to its place. I guess it's better to disable animation when resizing a window since it's affecting performance. Try to figure out possible ways of doing so.

**Comment by byzyk** [comment]
image

A bit more space on the sides for text copies :) It's fine for images/sliders though!

**Comment by v-nakon** [comment]
added header for mobile and search.

**Comment by byzyk** [comment]
Let's have a call later today?

File: src/style.css [hunk 1]
```diff
@@ -855,84 +856,23 @@ footer {
...
   }
-  
-  .menu-mobile {
-    width: 34px;
-    height: 23px;
-    cursor: pointer;
+
+  .container-toggle {
+    width: 100%;
+    right: -100%;
```
Comment (v-nakon) [inline src/style.css]: help me fix menu toggle. i created this style for mobile display, but it's not work.

Comment (byzyk) [reply src/style.css]: define "doesn't work" :)

Comment (byzyk) [reply src/style.css]: image

**Comment by v-nakon** [comment]
deleted `our-works-placeholders` for mobile, because when i deleted `transition` problem with img not solved.

**Comment by byzyk** [comment]
Good to go?

**Comment by v-nakon** [comment]
> Good to go?

yes

**Pull request merged**